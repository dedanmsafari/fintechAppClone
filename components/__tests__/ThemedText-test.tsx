import * as React from 'react';
import { Text } from 'react-native';

import renderer from 'react-test-renderer';

describe('Index Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Text>Hello World!</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
