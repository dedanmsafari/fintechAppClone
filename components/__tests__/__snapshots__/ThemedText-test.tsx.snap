// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Component renders correctly 1`] = `
<Text>
  Hello World!
</Text>
`;
